servicetalk-examples/grpc/protoc-options/src/main/java/io/servicetalk/examples/grpc/protocoptions/BlockingProtocOptionsClient.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2020 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.examples.grpc.protocoptions;
+
+import io.servicetalk.grpc.netty.GrpcClients;
+
+import io.grpc.examples.helloworld.GreeterSt.BlockingGreeterClient;
+import io.grpc.examples.helloworld.GreeterSt.ClientFactory;
+import io.grpc.examples.helloworld.HelloReply;
+import io.grpc.examples.helloworld.HelloRequest;
+
+public final class BlockingProtocOptionsClient {
+
+    public static void main(String[] args) throws Exception {
+        try (BlockingGreeterClient client = GrpcClients.forAddress("localhost", 8080)
```

Comment:
Consider using `GreeterSt.BlockingGreeterClient` and adding a comment that highlights the `St` suffix. Otherwise, it's hard to understand where the suffix is added. `import` section is usually collapsed in IDE. Something like:
```
// Greeter service is generated with "St" suffix:
try (GreeterSt.BlockingGreeterClient client = ...
```
Same comment for `BlockingProtocOptionsServer` class.

Reply:
done